storage/innobase/xtrabackup/test/suites/reducedlock/limit_file_handles.sh

```diff
@@ -0,0 +1,74 @@
+###############################################################################
+# PXB-3279: Remove requirement to have open file handles equal to files in datadir
+###############################################################################
+
+. inc/common.sh
+
+require_debug_pxb_version
+require_pro_pxb_version
+
+open_file_limit=$(ulimit -n)
+ulimit -n 1024
```

Comment:
Did you run the test on Jenkins? The user might not have permission to execute this.